blog-subPage/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Elvish | Blog</title>
+    <link rel="stylesheet" href="./css/reset.css">
+    <link rel="stylesheet" href="../css/font-awesome.min.css">
+    <link rel="stylesheet" href="./css/layout.css">
+    <link rel="stylesheet" href="./css/header.css">
+    <link rel="stylesheet" href="./css/aside.css">
+    <link rel="stylesheet" href="./css/custom.css">
+</head>
+<body>
+    <!-- header -->
+    <header class="container">
+        <div class="row">
+            <div class="header col-12"></div>
+        </div>
+    </header>
+    <!-- hero -->
+    <div class="container-fluid hero">
+        <div class="bg-overlay">
+            <div class="row">
+                <div class="hero-texts col-12">
+                    <h1>Latest news</h1>
+                    <span class="hero-links"><a href="#">Home</a> &sol; <a href="#">Blog</a></span>
+                </div>
+            </div>
+        </div>
+    </div>
+    <!-- main -->
+    <main class="container">
+        <div class="row">
+
+            <div class="container-fluid col-8 col-md-12">
+                <div class="container-fluid blog"></div>
+                <div class="blog-pages">
```

Comment:
Pagal Rimanto komentara container/container-fluid, row ir col klases negali buti taikomos toj pacioj eilutej (tam paciam dive/elemente) nes prisitaiko tik pirmoji - tarkim 36 eilutej galios container-fluid, o col-8 col-md-12 nebe. Col-8 col-md-12 turi taikyti vidiniam "vaikui"/div. Jei gerai suprantu 37 eilutej nereikia kartoti container fluid, nes ji paveldi is 36 eilutes.